.travis.yml

```diff
@@ -40,26 +38,19 @@ install:
     - ln -s $HOME/.local/share/pysap/pysap-data/pysap-data/* $HOME/.local/share/pysap
     - ls -l $HOME/.local/share/pysap
     - pip install --upgrade pip
-    - pip install matplotlib
     - pip install coverage nose pytest pytest-cov
     - pip install coveralls
     - pip install pycodestyle
-    - pip install pybind11 nibabel pyqt5 pyqtgraph astropy 
-    - pip install git+https://github.com/CEA-COSMIC/ModOpt.git
-    - pushd ../
-    - git clone https://github.com/CEA-COSMIC/pysap
-    - cd pysap
-    - python setup.py install
+    - pip install -b pysap_build --no-clean --no-deps git+https://github.com/cea-cosmic/pysap
```

Comment:
why don't we want the dependencies of `pysap`?

Reply:
Oh my bad, bad copy, changing this. I needed noclean

Reply:
And thats why the tests are failing